docs/nodejs/bootstrap.md

```diff
@@ -0,0 +1,64 @@
+
+## Configuration
+The server can be configured from the docker-compose.yml file.
+If no values are entered, the server will use a default configuration for testing.
+
+
+|          Variable          | Description                                                                                                                                                                                                                     |
+|:--------------------------:|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| SEED          | The seed of your Kin account                                                                                                                                                                                      |
+| HORIZON_ENDPOINT          | The URL for the Kin blockchain|
+| NETWORK_PASSPHRASE            | The passphrase for the Kin blockchain                                                                                                                                                                                                                                                                                                                                                                                       |
+| NETWORK_NAME          | The name of the selected network
+| APP_ID                 | The app ID (used to identify transactions)                                                                                                                                                                                                  |
+| CHANNEL_COUNT                 | How many channels to use. **See more in the Channels section**|                                                                                                                                                                                   |
+| CHANNEL_STARTING_BALANCE                | Initial Kin balance of each channel                                                                                                                        |
+| CHANNEL_SALT             | A string to be used when creating the channels                                                                                                                                                      |
+| PORT                | Port to serve the app on                                                                                                                                                                                                             |
+| LOG_LEVEL             | Log level, either "INFO" or "ERROR"                                                                                                                                                      |
+
+
+## Running the Server
+```bash
```

Comment:
must we clone the repo, right? if so, add this step.
In general, I think we need here 0 to 1 steps including how to generate a seed, and default values inside the docker-compose for reference. the idea is to have dead-simple instruction in one place.
I understand that it's borrowed from python bootstrap, so adding also @Ronserruya to get his input.